macOS/DuckDuckGo/DBP/VPNBypassServiceExtensions.swift

```diff
@@ -30,15 +31,7 @@ extension VPNBypassService {
     }
 
     public var isSupported: Bool {
-#if APPSTORE
-#if NETP_SYSTEM_EXTENSION
-        return true
-#else
-        return false
-#endif
-#else
-        return true
-#endif
+        VPNAppState(defaults: .netP).isUsingSystemExtension
```

Comment:
This is for DBP.  It should no longer look at `NETP_SYSTEM_EXTENSION` but `VPNAppState(defaults: .netP).isUsingSystemExtension`